### PR TITLE
fix(tooltip): Firefox overflowing text when you have a long enough string without spaces

### PR DESCRIPTION
**Pull request by bkrissIBM**
## Overview

While using Carbon Components React In Firefox, if strings get too long without spaces, the text will flow outside of the tooltip

screen shot 2018-04-27 at 3 31 46 pm

https://codesandbox.io/s/n4o33kvp8m

### Changed

Added css to .bx--tooltip

## Testing / Reviewing

Add the following string to a Tooltip and hover over in Firefox:
ibmcloud.public.iaas-telemetry.us-south.xxx01.0e8df11f-abcd-47a0-8468-12596966a59d

Tested in several versions of carbon-components-react up to version 5.44

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ When you're at a good stopping place and you're ready for feedback from other co
 > - `<scope>` The scope could be anything specifying place of the commit change or the thing(s) that changed.
 >
 > __Commit message format:__
-> ```
+```
```
Comment (asudoh) [inline docs/contributing.md:213]: Is the change here intentional…?

**Review by asudoh — COMMENTED** [review]
Nice fix - Thanks @bkrissIBM!

File: docs/contributing.md (hunk 1 above)
Comment (bkrissIBM) [reply docs/contributing.md:213]: Yes, I believe the 3 back-ticks are intended to be the beginning of a code block, but the arrow is breaking that. You can tell by looking at that doc in the browser. Removing the arrow fixed it on my end.

**Review by asudoh — APPROVED** [review]

**Review by tw15egan — APPROVED** [review]
Looks great! 👍 ✅

**Pull request merged**